packages/repository/src/entity/factory.test.ts

```diff
@@ -278,8 +359,10 @@ describe("Entity", () => {
       })
     })
 
-    it("Given entity, When dumping to regular object, Then return regular object with data", () => {
-      const { createEntity } = entityModelFactory<TestRawEntityData>()()
+    it("Given entity, When dumping to regular object, Then return regular object with data", ({
+      entitySchema,
+    }) => {
+      const { createEntity } = entityModelFactory({ schema: entitySchema })
       const data = {
         foo: "bar",
       }
```

Comment:
to aż się prosi wynieść, robi tylko niepotrzebny szum w testach

Reply:
Good catch, można to zastąpić przez `fakeData` z kontekstu 👍